static/styles/zulip.scss

```diff
@@ -1170,6 +1170,10 @@ td.pointer {
     color: hsl(200, 100%, 40%);
 }
 
+.m-l-10 {
```

Comment:
Can you find a better class name than this? I'm not sure what this is supposed to mean. I'd actually prefer a more general selector like `.message_failed i` for this.